tests/queries/0_stateless/01526_client_start_and_exit.expect

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/expect -f
+
+log_user 0
+set timeout 5
+match_max 100000
+
+if ![info exists env(CLICKHOUSE_PORT_TCP)] {set env(CLICKHOUSE_PORT_TCP) 9000}
+
+spawn clickhouse-client --port "$env(CLICKHOUSE_PORT_TCP)"
+expect ":) "
+send -- "\4"
```

Comment:
maybe add comment: "\4 = EOT (end of transmission), corresponds to pressing Ctrl+D"